CMakeLists.txt

```
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.5)
+project(zed_cpu)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

```suggestion
cmake_minimum_required(VERSION 3.10.2)
project(zed_cpu)

set(CMAKE_CXX_STANDARD 17)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```
Think we should go with c++17 for ROS2